README.md

```diff
@@ -103,6 +103,11 @@ available below.
 # Default: unset
 shell: bash
 
+# For the commands `kubie edit/edit-config`
+# Possible values: Any installed text editor
+# Default: ""
```

Comment:
I think default should be "unset"

Reply:
You are right, it is a documentation mistake I've done because it is actually unset.